Analytics/Integrations/Taplytics/SEGTaplyticsIntegration.m

```diff
@@ -94,6 +102,23 @@ - (NSString *)apiKey {
   return self.settings[@"apiKey"];
 }
 
+- (NSNumber *)delayLoad {
+    NSString *value = self.settings[@"delayLoad"];
+    
+    if (value != nil)
+        return [NSNumber numberWithInt:[value intValue]];
+    
+    return nil;
+}
+
+- (NSNumber *)shakeMenu {
+    NSString *value = self.settings[@"shakeMenu"];
```

Comment:
I think the default serialization parses booleans as numbers so you'll have to do:

```obj-c
if ([(NSNumber *)[self.settings objectForKey:@"shakeMenu"] boolValue]) {
    return YES;
}
return NO;
```